doc/tutorial/get_started.rst

```diff
@@ -385,6 +387,11 @@ See the full initialisation process here:
     Using 1 disk(s) on "micro3" for remote storage pool
 
    Would you like to set up CephFS remote storage? (yes/no) [default=yes]:  yes
+   Configure a dedicated Ceph network? (yes/no) [default=no]: yes
+   Choose either an IPv4 or IPv6 subnet (CIDR notation) to describe your Ceph dedicated cluster: 192.168.0.0/24
+   Interface "enp7s0" ("192.168.0.3") detected on cluster member "micro2"
+   Interface "enp7s0" ("192.168.0.4") detected on cluster member "micro3"
+   Interface "enp7s0" ("192.168.0.2") detected on cluster member "micro1"
```

Comment:
Wait, are we configuring it here or not?
Should the answer for line 274 be `no`?

Reply:
This bit needs to go, the two above questions represent the new usage.

Reply:
fixed